client/src/pages/MoviesList/index.js

```diff
@@ -1,18 +1,59 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 
 import { MovieListBackground } from '../../styles/Background'
 import Marquee from '../../components/Marquee'
 import Posters from './Posters'
 import SelectedMovie from './SelectedMovie'
+import TurnsTable from './TurnsTable'
+import { useHouses } from '../../providers/Houses'
+import { useTurns } from './utility'
+import { ReelImage } from '../../styles/LoadingReel'
 
 const MoviesList = () => {
   const [selectedMovie, setSelectedMovie] = useState()
 
+  const { currentHouse, userId } = useHouses()
+  const { getHouseTurns, houseTurns } = useTurns()
+
+  useEffect(() => {
+    if (currentHouse && !houseTurns) {
+      getHouseTurns(currentHouse.id)
+    }
+  }, [getHouseTurns, houseTurns, currentHouse])
+
+  if (!currentHouse || !houseTurns) {
+    return (
+      <MovieListBackground>
+        <ReelImage />
+      </MovieListBackground>
+    )
+  }
+
   return (
     <MovieListBackground>
-      <Marquee />
-      {!selectedMovie && <Posters setSelectedMovie={setSelectedMovie} />}
-      {selectedMovie && <SelectedMovie selectedMovie={selectedMovie} />}
+      <Marquee
+        currentTurn={houseTurns.current_turn.username}
+        nextTurn={
+          houseTurns.next_turn
```

Comment:
I think you can use null coalesce operator here. `houseTurns.next_turn || houseTurns.current_turn`. 
Basically, if the first value is null or undefined, then the second value will be chosen.